set.mm

```diff
@@ -34155,6 +34155,20 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
       ( cvv wcel cab wb nfcv elabgf ax-mp ) DHIDACJIBKFABCDHCDLEGMN $.
   $}
 
+  ${
+    $d A y $.  $d x y $.  $d ph y $.  $d ch y $.  $d ps x $.
+    elabgw.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    elabgw.2 $e |- ( y = A -> ( ps <-> ch ) ) $.
+    $( Membership in a class abstraction, using implicit substitution and an
+       intermediate setvar ` y ` to avoid ~ ax-10 , ~ ax-11 , ~ ax-12 .  It
+       also avoids a disjoint variable condition on ` x ` and ` A ` .  Compare
+       ~ sbievw2 .  (Contributed by SN, 20-Apr-2024.) $)
```

Comment:
Can you replace the last sentence by e.g.,
> This is to ~ elabg what ~ sbievw2 is to ~sbievw .

which is an economical way to link to these three theorems and explain their relationship.

Reply:
Done